### PR TITLE
CONCF-553 Add wikia stats module to Curated Main Page

### PR DESCRIPTION
**Pull request by bognix**
https://wikia-inc.atlassian.net/browse/CONCF-553
https://github.com/Wikia/app/pull/7366

### CODE REVIEW

File: front/templates/main/article.hbs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {{! FIXME this is a temporary solution, main pages should be decoupled from article}}
 {{#if model.isCuratedMainPage}}
 	<h1 class='article-title'>{{siteName}}</h1>
+	{{ wikia-stats model=model.mainPageData.wikiStats}}
```
Comment (rogatty) [inline front/templates/main/article.hbs]: Why not put it in the `main-page.hbs`?

**Comment by vforge** [comment]
I agree with @rogatty 's comment, otherwise LGTM

File: front/templates/main/components/main-page.hbs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <section class='main-page-body mw-content'>
+	{{ wikia-stats model=model.wikiStats}}
```
Comment (rogatty) [inline front/templates/main/components/main-page.hbs]: Let's be consistent and use `{{#if model.wikiStats}}`

**Comment by vforge** [comment]
:heavy_check_mark: LGTM

**Comment by rogatty** [comment]
LGTM

File: front/styles/main/component/_wikia-stats.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
+.wikia-stats {
+	text-align: center;
+
+	.wikia-stats-container {
```
Comment (hakubo) [inline front/styles/main/component/_wikia-stats.scss]: does it have to be a child?

File: front/templates/main/components/wikia-stats.hbs [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<div class="wikia-stats-container">
+	{{#if model.pages}}
+		<div class="wikia-stats-item">
```
Comment (hakubo) [inline front/templates/main/components/wikia-stats.hbs]: this looks like a partial to me. what do you think?

**Comment by hakubo** [comment]
LGTM

**Comment by rogatty** [comment]
LGTM

**Comment by rogatty** [comment]
@hakubo related to your commit `We use ' everywhere` (https://github.com/Wikia/mercury/commit/bd03336ceedad447a949d833f0a178280d9b32b4): https://github.com/Wikia/guidelines/pull/80

**Pull request merged**